### PR TITLE
Remove Edit Button

### PR DESCRIPTION
**Pull request by clmedders**
## Summary

This Pr will remove the edit button that lives on the site that allows users the ability to suggest edits via github 

This will also remove the other edit button at the end of the pages along with the `last updated` functionality. 

Will also update the footer link to `Submit issue on GitHub` and link to the issues page for users to make and issue.

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-remove-edit-btn/)

### Solution

- Removed template page for the edit button 
- Removed both styles and js files that were associated with this button 
- Removed the forward in the global styles sheet 

### How To Test

1. Visit [preview link](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-remove-edit-btn/) and ensure that button is no longer present 
2. Also visit [other preview link]() can scroll to the bottom and check that edit and last updated are no longer present. 
3. Check that footer link now reads `Submit issue on GitHub` and link takes user to issues page. 
4.  Check that any mention of the edit button is removed from the repo 

---

### Dev Checklist

- [ ] PR has correct labels
- [ ] A11y testing (voice over testing, meets WCAG, run axe tools)
- [ ] Code is formatted properly

tagging supervisor for reference - @rayestrada

### CODE REVIEW

**Review by mejiaj — CHANGES_REQUESTED** [review]
Looks good for the most part. Found some old references in template we should remove to avoid any confusion.

- [x] References to css class `gh-edit-tools` are gone.
- [ ] References used in JS are gone.

I still see references to `data-gh-edit-page` in templates.

**Review by nick-mon1 — CHANGES_REQUESTED** [review]

File: themes/digital.gov/src/js/gh-edit-page.js [hunk 1]
```diff
@@ -1,87 +0,0 @@
-// Sets the github URL link and button styles for each resource when edit-mode is enabled
-// eslint-disable-next-line func-names
-(function () {
-  const githubEditLinks = document.querySelectorAll("[data-gh-edit-page]");
```
[the remark this answers is not in the document]
Comment (clmedders) [reply themes/digital.gov/src/js/gh-edit-page.js:4]: updated and removed

**Review by nick-mon1 — DISMISSED** [review]
What I Tested:

- [x] tested "Submit Github issue" link
- [x] removed `$path` when not used in templates/component

**Review by nick-mon1 — DISMISSED** [review]
Approved

**Review by nick-mon1 — APPROVED** [review]

**Review by mejiaj — APPROVED** [review]
Approved.

I tested the following:
-  [x] References to `data-gh-edit-page` are gone.
-  [x] References to `gh-commit-info` are gone.
-  [x] No issues with buildJS task.
-  [x] No visual regressions.

**Pull request merged**